.github/workflows/ci.yml

```diff
@@ -13,24 +13,56 @@ on:
 
 env:
   DOC_TROUBLESHOOTING: "Warnings like 'reference target not found' occur when Sphinx tries to interpret text as a Python object, but is unable to do so. This warning can often be fixed by surrounding text in double back ticks instead of single back ticks (e.g., by changing `y` to ``y``) so that it gets formatted as an in-line literal. For more information about addressing documentation build failures, please check out the documentation troubleshooting guide at: https://docs.plasmapy.org/en/latest/contributing/doc_guide.html#troubleshooting"
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
 
 
   tests:
     name: ${{ matrix.name }}
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
```

Comment:
There were some problems with GitHub Actions today, and our doc build was left hanging for close to an hour.  Since none of the workflows in this file typically last more than 5 minutes, it seemed reasonable to have a timeout that's much shorter than GitHub's default of 360.